scvi/module/_mrdeconv.py

```diff
@@ -43,8 +43,14 @@ class MRDeconv(BaseModuleClass):
         Mean parameter for each component in the empirical prior over the latent space
     var_vprior
         Diagonal variance parameter for each component in the empirical prior over the latent space
+    weight_vprior
+        Weight parameter for each component in the empirical prior over the latent space
     amortization
         which of the latent variables to amortize inference over (gamma, proportions, both or none)
+    l1_sparsity
```

Comment:
call it l1_reg?
scalar parameter indicating the strength of the L1 penalization